docs/csharp/tutorials/intro-to-csharp/branches-and-loops-local.md

```diff
@@ -1,18 +1,18 @@
 ---
-title: Branches and loops tutorial - C# local quickstarts
-description: In this quickstart about branches and loops, you write C# code to explore the language syntax that supports conditional branches and loops to execute statements repeatedly.
+title: Branches and loops tutorial - C# introduction tutorial
+description: In this tutorial about branches and loops, you write C# code to explore the language syntax that supports conditional branches and loops to execute statements repeatedly.
 ms.date: 10/31/2017
 ms.custom: mvc
 ---
 # Branches and loops
```

Comment:
one more nit about mismatch of H1 and the "title" in metadata

Reply:
per earlier comment, leaving these different.